app/[lang]/dictionaries/en.json

```diff
@@ -5,8 +5,47 @@
     "receiveFloodWarnings": "Receive flood warnings",
```

Comment:
Could the dicts just be regular TypeScript objects instead of a JSON-file? We could the type them with `Dict` to make sure we're not leaving out any fields